plugin.video.tyt/pyjwt/AUTHORS

```diff
@@ -0,0 +1,29 @@
+PyJWT lead developer
```

Comment:
this would be better as a script.module if you have not introduced any changes to the module

Reply:
Also note the library itself may be licensed under a different license than your plugin

Reply:
I would like to do this.  Is there somewhere you can point me to that will give an example of how to do this?

Thanks For the help

Reply:
Yes thats it. Then you just import it as a dependency of this addon

Reply:
Ok, merged script.module.pyjwt to repo-scripts.  I remove the pyjwt folder from my plugin, and added the script.module.pyjwt import to my addon.xml.  I guess I'll wait until the repo-scripts to propogate so I can test to make sure everything works, then I'll amend my commit with the updated code.

Reply:
@lottaboost did you end up commiting the changes to your branch (this pr)? There's only a single commit from 2 days ago.

Reply:
Not yet, going to do now.  Wanted to wait till the library propagated and repo-scripts, that way people don't think the plugin updated to a broken plugin that is missing a dependency.

Reply:
I amended the commit.  It is now ready for merge, pending your approval.
Thanks